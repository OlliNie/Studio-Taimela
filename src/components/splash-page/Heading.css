.heading{
  color:#FFD67A;
  margin:0 0 1rem 0;
  font-weight: bold;
  font-size:1.5rem;
  @media(min-width: 420px){
    font-size:2rem;
  }
  @media(min-width: 520px){
    font-size:2.5rem;
  }
 
  @media(min-width:900px){
   font-size:3rem;
  }
  @media(min-width:1100px){
    font-size: 2.1rem;
  }
  @media(min-width:1300px){
    font-size: 3rem;
  }
  @media(min-width:1600px){
    font-size: 4rem;
  }
 
}