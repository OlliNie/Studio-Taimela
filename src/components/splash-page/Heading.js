import React from 'react';
import { headingLines } from '../../text/splashPage';
import styles from './Heading.css';

export default function Heading() {
  const heading = headingLines.map((line, i) => (
    <li key={i}>
      <p>{line}</p>
    </li>
  ));
  return <ul className={styles.heading}>{heading}</ul>;
}
