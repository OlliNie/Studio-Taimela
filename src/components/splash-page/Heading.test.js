import React from 'react';
import { shallow } from 'enzyme';
import Heading from './Heading';

describe('Heading component', () => {
  it('renders ContactPage', () => {
    const wrapper = shallow(<Heading />);
    expect(wrapper).toMatchSnapshot();
  });
});
