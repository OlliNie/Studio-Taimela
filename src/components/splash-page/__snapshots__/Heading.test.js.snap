// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading component renders ContactPage 1`] = `
<ul
  className="heading"
>
  <li
    key="0"
  >
    <p>
      STRATEGIC MARKETER.
    </p>
  </li>
  <li
    key="1"
  >
    <p>
      CREATIVE THINKER.
    </p>
  </li>
  <li
    key="2"
  >
    <p>
      TECH ENTHUSIAST.
    </p>
  </li>
</ul>
`;
